docs/API/schema_v0.0.3.json

```diff
@@ -0,0 +1,1252 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LORIS API",
+    "description": "This document specifies the LORIS REST API.",
+    "version": "0.0.3"
+  },
+  "host": "alex-dev.loris.ca",
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "basePath": "/api/v0.0.3",
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/login": {
+      "post": {
+        "description": "Users can be be authenticated using JSON Web Tokens by sending a POST request with their username and password",
+        "tags": [
+          "Login"
+        ],
+        "operationId": "Authentication",
+        "parameters": [
+          {
+            "name": "data",
+            "required": true,
+            "description": "Request payload",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "username": {
+                  "type": "string"
+                },
+                "password": {
+                  "type": "string",
+                  "format": "password"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An object with an authenticity token",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "token": {
+                  "type": "string",
+                  "description": "JWT token"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          }
+        }
+      }
+    },
+    "/projects": {
+      "get": {
+        "description": "Return a list of the project names in this Loris instance",
+        "tags": [
+          "Project"
+        ],
+        "security": [
+          {
+            "Bearer": []
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "an array of projects",
+            "schema": {
+              "type": "array",
```

Comment:
We should never do that. Array constructor can be overridden or its prototype setter. Security risk there